quarkus/service/src/test/java/org/apache/polaris/service/quarkus/ratelimiter/RateLimiterFilterTest.java

```diff
@@ -77,7 +77,11 @@ public Map<String, String> getConfigOverrides() {
           "polaris.metrics.tags.environment",
           "prod",
           "polaris.realm-context.type",
-          "test");
+          "test",
```

Comment:
This test would fail without this change because each realm would have a different pair of RSA keys. So switching to shared secret.